tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -160,6 +160,12 @@ class HarnessCase(unittest.TestCase):
     started = 0
     """number: test case started timestamp"""
 
+    case_need_shield = False
```

Comment:
Can we infer `case_need_shield` from `device_order` ? `case_need_shield` seems redundant.

Reply:
Yes, I understand. From my original intention, these two parameters could be more legible. And if there is no drag order but need shield DUT like med_6_3_2, just only need to define case_need_shield = True